README.md

```diff
@@ -100,6 +100,8 @@ docker run -v "$(pwd)":/app -w /app kdf-build-container cargo build
 
 Just like building it on your host system, you will now have the target directory containing the build files.
 
+Alternatively, container images are available on [DockerHub](https://hub.docker.com/r/komodoofficial/komodo-defi-framework)
```

Comment:
btw, the readme of this dockerhub page is also outdated. will it sync automatically with github?

Reply:
It should